nodeJS/getting-started/Getting-Started.md

```diff
@@ -29,6 +29,13 @@ By the end of this lesson, you should be able to do the following:
   - The File Uploads and Email lessons are optional at this point but it might be useful to at least skim them so you can know where to look if you need that functionality.
 </div>
 
+
```

Comment:
The placement of this additional resources section is not consistent with the placement of the section in other lessons in our curriculum. It will need to be moved to the appropriate place, typically the bottom.